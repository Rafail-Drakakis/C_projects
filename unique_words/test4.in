Areas such as operating systems, networks and embedded systems investigate the principles and design behind complex systems. Computer architecture describes the construction of computer components and computer-operated equipment. Artificial intelligence and machine learning aim to synthesize goal-orientated processes such as problem-solving, decision-making, environmental adaptation, planning and learning found in humans and animals. Within artificial intelligence, computer vision aims to understand and process image and video data, while natural language processing aims to understand and process textual and linguistic data.
The fundamental concern of computer science is determining what can and cannot be automated.[7][8][9][10][11] The Turing Award is generally recognized as the highest distinction in computer science.[12][13]
