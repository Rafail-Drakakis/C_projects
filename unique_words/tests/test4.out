Areas 1
such 2
as 3
operating 1
systems, 1
networks 1
and 11
embedded 1
systems 1
investigate 1
the 3
principles 1
design 1
behind 1
complex 1
systems. 1
Computer 1
architecture 1
describes 1
construction 1
of 2
computer 4
components 1
computer-operated 1
equipment. 1
Artificial 1
intelligence 1
machine 1
learning 2
aim 1
to 3
synthesize 1
goal-orientated 1
processes 1
problem-solving, 1
decision-making, 1
environmental 1
adaptation, 1
planning 1
found 1
in 2
humans 1
animals. 1
Within 1
artificial 1
intelligence, 1
vision 1
aims 2
understand 2
process 2
image 1
video 1
data, 1
while 1
natural 1
language 1
processing 1
textual 1
linguistic 1
data. 1
The 2
fundamental 1
concern 1
science 1
is 2
determining 1
what 1
can 1
cannot 1
be 1
automated.[7][8][9][10][11] 1
Turing 1
Award 1
generally 1
recognized 1
highest 1
distinction 1
science.[12][13] 1
